tsup.config.ts

```diff
@@ -1,11 +1,18 @@
 import { defineConfig } from "tsup"
+import { cache } from "esbuild-plugin-cache"
 
 export default defineConfig({
-  entry: ["src/index.ts", "src/cli/**/*.ts"],
+  entry: [
+    "src/index.ts",
+    "src/cli/index.ts",
+    "src/cli/entry.ts",
+    "src/cli/browser.ts",
+  ],
   dts: true,
   sourcemap: true,
   format: ["cjs", "esm"],
   env: {
     NODE_ENV: "production",
   },
+  esbuildPlugins: [cache({})],
```

Comment:
I spent far too long faffing around with Webpack before I realized there was an ESBuild plugin for `http` imports... 😐